pkg/functionconfig/types.go

```diff
@@ -299,6 +299,14 @@ func (s *Spec) GetHTTPPort() int {
 			httpPort, httpPortValid := trigger.Attributes["port"]
 			if httpPortValid {
 				switch typedHTTPPort := httpPort.(type) {
+				case int8:
```

Comment:
Can do all these with one case, coma delimited
```
case int8, int16, int32, int64, uint64:
```
Also what about uint8, uint16, uint32 ?

Reply:
not for type assertion
when you combine `case x, y:`
the case value becomes `interface{}`